### PR TITLE
Update TOC members: add dsimansk and zroubalik

### PR DESCRIPTION
**Pull request by zroubalik**
# Changes
- adding @dsimansk and @zroubalik to TOC members
- also removing @rhuss  :(

### CODE REVIEW

File: groups/committee-oversight/groups.yaml [hunk 1]
```diff
@@ -9,10 +9,10 @@ groups:
...
     owners:
-      - rhuss@knative.team
-      - rhuss@redhat.com
       - evankanderson@knative.team
       - evana@vmware.com
       - evan.k.anderson@gmail.com
       - dprotaso@gmail.com
       - n3wscott@knative.team
+      - zroubalik@gmail.com
+      - zroubali@redhat.com
```
Comment (zroubalik) [inline groups/committee-oversight/groups.yaml:18]: @dsimansk please let me know which email you'd like to add in here

Comment (dsimansk) [reply groups/committee-oversight/groups.yaml:18]:
```suggestion
      - zroubali@redhat.com
      - dsimansk@redhat.com
      - d.simansky@gmail.com
```
Thanks!

**Comment by dprotaso** [comment]
cc @upodroid  looks like a bug in the groups tool?

**Comment by upodroid** [comment]
you bumped the go version, i'll need to take a look

**Comment by upodroid** [comment]
Identified the cause https://github.com/prometheus/procfs/pull/431 will push a fix in a separate PR.

**Comment by vaikas** [comment]
/test pull-knativeteam-groups-test

Hoping now that #1095 has been merged things will work :)

**Pull request merged**